src/ascii_str.rs

```diff
@@ -1065,6 +1069,9 @@ impl AsAsciiStr for CStr {
 #[cfg(test)]
 mod tests {
     use super::{AsAsciiStr, AsAsciiStrError, AsMutAsciiStr, AsciiStr};
+    use alloc::string::{String, ToString};
```

Comment:
There is a `cfg` annotation missing here:
```suggestion
    #[cfg(feature = "alloc")]
    use alloc::string::{String, ToString};
```